src/main/java/InputProcess.kt

```diff
@@ -0,0 +1,278 @@
+import BechTools.BASE64_FORMAT
+import BechTools.BINARY_FORMAT
+import BechTools.DEC_FORMAT
+import BechTools.HEX_FORMAT
+import BechTools.NO_FORMAT
+
+// Written by Mikita 7.4.2022
+/*
+When function returned 'false' – it means FAIL
+'true' – successful result
+ */
+
+object InputProcess {
+
+    private var isToEncode: Boolean = false
+    private var isToDecode: Boolean = false
+    private var isPrintIntoFile: Boolean = false
+
+    private var inputString: String = ""
+    private var outputFilePath: String = ""
+
+    private var inputFormat: Int = NO_FORMAT // Default: DEC for encode // BASE64 for decode
+    private var outputFormat: Int = NO_FORMAT //Default: BASE64 for encode // DEC for decode
```

Comment:
Inconsistent comment formatting: //_Default vs. //Default. Also comment inside of comment is not conventional.